ipfs-cluster-service/main.go

```diff
@@ -434,3 +531,42 @@ func makeConfigs() (*config.Manager, *ipfscluster.Config, *rest.Config, *ipfshtt
 	cfg.RegisterComponent(config.Informer, numpinInfCfg)
 	return cfg, clusterCfg, apiCfg, ipfshttpCfg, consensusCfg, monCfg, diskInfCfg, numpinInfCfg
 }
+
+func makeDummyHost(cfg *ipfscluster.Config) (host.Host, error) {
+	ps := peerstore.NewPeerstore()
+	for _, m := range cfg.Peers {
+		pid, decapAddr, err := ipfscluster.MultiaddrSplit(m)
+		if err != nil {
+			return nil, err
+		}
+		ps.AddAddr(pid, decapAddr, peerstore.PermanentAddrTTL)
+	}
+	pid, decapAddr, err := ipfscluster.MultiaddrSplit(cfg.ID)
+	if err != nil {
+		return nil, err
+	}
+	ps.AddAddr(pid, decapAddr, peerstore.PermanentAddrTTL)
+	
+	network, err := swarm.NewNetwork(
+		context.Background(),
+		[]ma.Multiaddr{cfg.ListenAddr},
+		cfg.ID,
+		ps,
+		nil,
+	)
+	if err != nil {
+		return nil, err
+	}
+		
+	bhost := basichost.New(network)
+	return bhost, nil
+}
+	
+func encodeState(state mapstate.MapState) ([]byte, error) {
```

Comment:
This is just copy pasted from libp2p-raft/codec.  I'd like your opinion on whether this should be exported from the module or kept inline here.